### PR TITLE
don't use deprecated request.REQUEST in cloudcare and apis

### PR DESCRIPTION
**Pull request by nickpell**
plus some import organizing

:blowfish:

### CODE REVIEW

File: corehq/apps/cloudcare/views.py [hunk 1]
```diff
@@ -245,14 +246,16 @@ def form_context(request, domain, app_id, module_id, form_id):
 
 
 def get_cases_vary_on(request, domain):
+    request_params = request.GET if request.method == 'GET' else request.POST
```
Comment (benrudolph) [inline corehq/apps/cloudcare/views.py]: pretty sure these are all GET requests, but this is probably safer. might be interesting to add a soft assert

File: corehq/apps/api/resources/v0_2.py [hunk 2]
```diff
@@ -2,16 +2,22 @@
```
Comment (benrudolph) [inline corehq/apps/api/resources/v0_2.py]: :clap:

File: corehq/apps/cloudcare/views.py (hunk 1 above)
Comment (czue) [inline corehq/apps/cloudcare/views.py]: @nickpell I know you didn't want to add a helper method though it's not clear to me that having this line in a bunch of views is any better.

what if we had a function that was: `request_params = expect_GET(request)` that returns `request.GET`, but then soft asserts if it's a POST and returns `request.POST`. then we could put what we think it is everywhere and once there weren't any assertions we could fully switch over

Comment (nickpell) [inline corehq/apps/cloudcare/views.py]: excellent approach, will do

Comment (nickpell) [inline corehq/apps/cloudcare/views.py]: https://github.com/dimagi/commcare-hq/pull/10807

**Pull request merged**